src/Bridge/Symfony/SonataFormBundle.php

```diff
@@ -11,21 +11,13 @@
  * file that was distributed with this source code.
  */
 
-namespace Sonata\Form\Bridge\Symfony\Bundle;
+namespace Sonata\Form\Bridge\Symfony;
 
 use Sonata\Form\Bridge\Symfony\DependencyInjection\SonataFormExtension;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 final class SonataFormBundle extends Bundle
 {
-    /**
-     * @return string
-     */
-    public function getPath()
```

Comment:
Does it make sense to move this class, where it is on 1.x?

Reply:
I add this file in 1.x too: 
https://github.com/wbloszyk/form-extensions/blob/0cb26479c51d9ae22c26600361e46013cbab595b/src/Bridge/Symfony/SonataFormBundle.php

We should move it like in twig-extensions.